pom.xml

```diff
@@ -92,6 +92,7 @@
 		<module>struts2-jquery-grid-showcase</module>
 		<module>struts2-jquery-mobile-showcase</module>
 		<module>struts2-jquery-archetypes</module>
+		<module>struts2-jquery-plugin-tests</module>
```

Comment:
Thanks :)

Regarding additional tests & modules, I think for now it's ok to put them in `struts2-jquery-integration-tests` - it will simplify configuration & maintenance burden :)